### PR TITLE
sea3d revision 2 + morph example

### PR DESCRIPTION
**Pull request by sunag**
Hi. I am updating all examples standardize all sea3d objects for a specifies namespace `THREE.SEA3D`, by my analyzed it has no injection over the native codes in three.js as the suggestion of @dubejf 

```
THREE.SEA3D.Object3D // SEA3D Object3D or Container3D
THREE.SEA3D.Mesh // SEA3D Mesh + Morph Helpers
THREE.SEA3D.SkinnedMesh // SEA3D SkinnedMesh + Skinned Animation Helpers
THREE.SEA3D.StandardMaterial // SEA3D Standard Material
THREE.SEA3D.Animation // SEA3D Animation + Events
THREE.SEA3D.AnimationHandler // SEA3D Animations Update
```

Also replaces the `TrackballControls` to `OrbitControls` as the suggestion of @WestLangley 

Add sound filter and getter functions in `Audio.js`. `webgl_loader_sea3d_sound.html` show the use of sound filters.

### CODE REVIEW

File: examples/webgl_loader_sea3d_morph.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+			var clock = new THREE.Clock();
+
+			function animate() {
+
+				var delta = clock.getDelta();
+				
+				requestAnimationFrame( animate );
+
+				controls.update();
```
Comment (WestLangley) [inline examples/webgl_loader_sea3d_morph.html]: With `OrbitControls` you should not have to call `controls.update()`, unless you are using the `autoRotate` feature.

Comment (sunag) [inline examples/webgl_loader_sea3d_morph.html]: Thanks!

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Is `webgl_loader_sea3d_hierarchy` displaying correctly? The legs seem to be unattached...?

screen shot 2015-09-02 at 21 51 07

**Comment by sunag** [comment]
Hi. Since then I did a optimized version for Three.JS, I will make a PR in a few minutes, with this correction as well.

http://community.poonya.com/160

Are you interested in more two examples? I say this because I do not know if there is any limit for this.

1. Vertex Animation
http://sunag.github.io/sea3d/Examples/Programmer/WebGL/Three.JS/Basics/vertex-animation.html

2. Physics incorporated in SEA3D with Ammo.js

Cheers.

**Pull request merged**